README.windows.md

```diff
@@ -436,3 +436,10 @@ running the resulting installer.
    inspect the list of loaded DLLs for bash, mintty, or another persistent
    process used to drive the build. Essentially *any* DLL outside of the Windows
    System directory is potential BLODA.
+
+   ### Compiling for Bash on windows
+   Bash on windows is an unsupported platform. However, compilation of julia is possible and functional. The packages required differ  from standard ubuntu compilation as some packages are not installed by default in the bash on windows ubuntu image.
+
+   The missing packages are `byacc texinfo python-minimal automake`. With these installed and the standard ubuntu packages dependancies and julia currently compile without issue. The line below can be copied into your terminal to install all required packages.
```

Comment:
All those dependencies were sourced as Julia compiled.
byacc is needed for gmp, I can't remember where the other two were called but I started on a clean bash on windows install for this and those were the packages it told me mid compilation were missing